### PR TITLE
Phlogiston no longer ignites every tick

### PR DESCRIPTION
**Pull request by SconesC**
Rehash of #10693 from @Iamgoofball with Phil's fix regarding negative fire stacks implemented 

This was made because as of writing, the comment from @phil235 was responded to 23 days ago but the issue was never fixed, preventing the original PR from being merged.

### CODE REVIEW

**Comment by daman997** [comment]
praise scones

**Comment by Shapsy** [comment]
:+1:

**Comment by Core0verload** [comment]
Point of flog is to ignite mob every tick. If you want to nerf it, make it ignite mobevery second tick.

**Comment by SconesC** [comment]
Please read the relevant discussions that led up to this change before making comments about it

**Comment by Miauw62** [comment]
i dont play chemist anymore so i dont really know how op phlog is from personal experience but

>join sybil
>say hi in ooc
>people see that im a coder
>ooc: ayy mclmao: nerf phlog pls

so i guess this is probably ok

**Comment by Robustin** [comment]
I'll give you an idea Miauw:

>Chemist throws bottle of phlog at HoS
>HoS burns to death despite constant fire extinguishering from 2 extinguishers and a borg extinguisher and an ointment patch
>Bring HoS to medbay after a few minutes and burn patch the husked corpse (this is also a stupid "feature" but not the right PR to remove it) and defibbing it.
>Defib is successful, HoS is revived and immediately bursts into flames, an additional burn patch and ointment cannot stave off the inevitable second death (cryo had no power).

**Comment by Core0verload** [comment]
Then just nerf bottles throwing.

**Comment by SconesC** [comment]
Throwing is a problem on it's own but phlogiston is arguably the most requested change I've seen in the past week of being a very very regular admin

Goof's fix wasn't getting anywhere

**Comment by Cheridan** [comment]
Thanks for cleaning up after goofball

File: code/modules/reagents/Chemistry-Reagents/Pyrotechnic-Reagents.dm [hunk 1]
```diff
@@ -128,13 +128,16 @@
...
+	..()
 
 /datum/reagent/phlogiston/on_mob_life(mob/living/M)
 	M.adjust_fire_stacks(1)
-	M.IgniteMob()
-	M.adjustFireLoss(0.2*M.fire_stacks)
+	var/burndmg = max(0.3*M.fire_stacks, 0.3)
+	M.adjustFireLoss(burndmg)
 	..()
-	return
```
Comment (Cheridan) [inline code/modules/reagents/Chemistry-Reagents/Pyrotechnic-Reagents.dm]: god how did we even let that in

**Comment by PKPenguin321** [comment]
heyooooooooo

**Comment by Iamgoofball** [comment]
:+1:

**Pull request merged**